README.md

```diff
@@ -0,0 +1,36 @@
+# Vendor Canonical Interfaces
```

Comment:
The title should definitely include Anbox

Reply:
+1, It makes sense to me